fpdf/fpdf.py

```diff
@@ -4458,6 +4485,7 @@ def _out(self, s):
         self.pages[self.page].contents += s + b"\n"
 
     @check_page
+    @support_deprecated_txt_arg
```

Comment:
Why should we support `txt=` there?

I don't think those methods every allowed for a `txt=` parameter before,
so I don't see the point of allowing it now, when we want to deprecate it!

Reply:
In the case of `interleaved2of5()`, #903 says otherwise. 😉

Reply:
Oh!

Indeed, my mistake 🤦‍♂️

Same for `code39()`, it used to accept a `txt` parameter:
https://github.com/py-pdf/fpdf2/blob/2.4.0/fpdf/fpdf.py#L3162

So I missed that in #903?! My bad...